packages/playwright-test/src/util.ts

```diff
@@ -307,16 +307,17 @@ function folderIsModule(folder: string): boolean {
   return require(packageJsonPath).type === 'module';
 }
 
-export function experimentalLoaderOption() {
-  return ` --no-warnings --experimental-loader=${url.pathToFileURL(require.resolve('@playwright/test/lib/transform/esmLoader')).toString()}`;
+export function execArgvWithExperimentalLoaderOptions() {
+  return [
+    ...process.execArgv,
+    '--no-warnings',
+    `--experimental-loader=${url.pathToFileURL(require.resolve('@playwright/test/lib/transform/esmLoader')).toString()}`,
+  ];
 }
 
-export function envWithoutExperimentalLoaderOptions(): NodeJS.ProcessEnv {
-  const substring = experimentalLoaderOption();
-  const result = { ...process.env };
-  if (result.NODE_OPTIONS)
-    result.NODE_OPTIONS = result.NODE_OPTIONS.replace(substring, '').trim() || undefined;
-  return result;
+export function execArgvWithoutExperimentalLoaderOptions() {
+  const execArgv = execArgvWithExperimentalLoaderOptions();
+  return process.execArgv.filter(arg => !execArgv.includes(arg));
```

Comment:
This is going to remove everything from `process.execArgv` because `execArgvWithExperimentalLoaderOptions()` includes `process.execArgv` inside.